src/components/users/Container.js

```diff
@@ -3,26 +3,12 @@ import { Route } from 'react-router-dom'
 
 import List from './List/List'
 import PostsContainer from '../posts/Container'
+import PostData from '../../api/posts'
 
 export default class Container extends React.Component {
   constructor (props) {
     super(props)
-    this.state = {
-      users: [
-        {
-          _id: '5de4',
-          username: 'example.user',
-          posts: [
-            {
-              _id: '6cj2',
-              content: 'This is an example post.',
-              emotion: 'joy',
-              created_at: new Date('2019-07-01')
-            }
-          ]
-        }
-      ]
-    }
+    this.state = PostData
```

Comment:
The app is crashing because it attempts to access an undefined variable 'users' in the list component. Here you are referencing the PostData but remember its good practice to load data in one of the component lifecycle functions such as componentDidMount(). Of course this would also require us to have a loading flag and hold rendering till the data is loaded. 
We already loaded users in the app component when logging in so It may be a good idea to catch posts at that point and pass it down as a prop. it is up to you how you wish to structure your app as to which component handles what. 
I'd recomend looking at the solutions branch : 

https://github.com/pce-uw-jscript400/w6-frontend/tree/solution-deployment
https://github.com/pce-uw-jscript400/w6-backend/tree/solution-deployment